SuperBuild/External_tbb.cmake

```
@@ -29,129 +29,59 @@ if((NOT DEFINED TBB_DIR
     )
     AND NOT Slicer_USE_SYSTEM_${proj})
 
-  set(tbb_ver "2021.5.0")
-  if (WIN32)
-    set(tbb_file "oneapi-tbb-${tbb_ver}-win.zip")
-    set(tbb_sha256 "096c004c7079af89fe990bb259d58983b0ee272afa3a7ef0733875bfe09fcd8e")
-  elseif (APPLE)
-    set(tbb_file "oneapi-tbb-${tbb_ver}-mac.tgz")
-    set(tbb_sha256 "388c1c25314e3251e38c87ade2323af74cdaae2aec9b68e4c206d61c30ef9c33")
-  else ()
-    set(tbb_file "oneapi-tbb-${tbb_ver}-lin.tgz")
-    set(tbb_sha256 "74861b1586d6936b620cdab6775175de46ad8b0b36fa6438135ecfb8fb5bdf98")
-  endif ()
-
-  # When updating the version of tbb, consider also
-  # updating the soversion numbers hard-coded below in the
-  # "fix_rpath" macOS external project step.
-  #
-  # To find out if the soversion number should be updated,
-  # inspect the installed libraries and/or review the value
-  # associated with
-  # (1) __TBB_BINARY_VERSION in include/oneapi/tbb/version.h
-  #     for libtbb
-  # (2) TBBMALLOC_BINARY_VERSION variable in the top-level
-  #     CMakeLists.txt for libtbbmalloc
-
-  if(APPLE)
-    set(tbb_cmake_osx_required_deployment_target "10.15") # See https://github.com/oneapi-src/oneTBB/blob/master/SYSTEM_REQUIREMENTS.md
-
-    if(NOT DEFINED CMAKE_OSX_DEPLOYMENT_TARGET)
-      message(FATAL_ERROR "CMAKE_OSX_DEPLOYMENT_TARGET is not defined")
-    endif()
-    if(NOT CMAKE_OSX_DEPLOYMENT_TARGET VERSION_GREATER_EQUAL ${tbb_cmake_osx_required_deployment_target})
-      message(FATAL_ERROR "TBB requires macOS >= ${tbb_cmake_osx_required_deployment_target}. CMAKE_OSX_DEPLOYMENT_TARGET currently set to ${CMAKE_OSX_DEPLOYMENT_TARGET}")
-    endif()
+  if(NOT DEFINED git_protocol)
+      set(git_protocol "https")
+  set(tbb_ver "v2022.2.0-rc1")
   endif()
 
-  #------------------------------------------------------------------------------
-  set(TBB_INSTALL_DIR "${CMAKE_BINARY_DIR}/${proj}-install")
+  ExternalProject_SetIfNotDefined(
+     Slicer_${proj}_GIT_REPOSITORY
+     ${git_protocol}://github.com/oneapi-src/oneTBB.git
```

To reflect the latest organization name that it redirects to now:
```suggestion
     ${git_protocol}://github.com/uxlfoundation/oneTBB.git
```
https://www.linuxfoundation.org/press/announcing-unified-acceleration-foundation-uxl